lambda/translmod.ml

```diff
@@ -1669,22 +1681,32 @@ let print_cycle ppf cycle =
     (Ident.name @@ fst @@ List.hd cycle)
 (* we repeat the first element to make the cycle more apparent *)
 
+let reason_to_message = function
+  | Unsafe_module_binding -> "defines an unsafe module"
+  | Unsafe_functor -> "defines an unsafe functor"
+  | Unsafe_typext -> "defines an unsafe extension constructor"
+  | Unsafe_non_function -> "defines an unsafe value"
```

Comment:
`"defines an unsafe"` is a common prefix in all cases, I think this part could be moved to the caller, and then this function would do something fairly clear/simple, which is to name the thing that is unsafe. (Currently it is less clear what it does, and as a result the function name is generic and unclear.)

Reply:
The function is an anti-pattern, error message should not be split into text fragment that cannot be understood without more context. A good litmus test is "Can I translate the error message?" without contextual information. Here, the answer is no (because of at least subject-verb-object order and subject-verb concordance).